FordLabs/PeopleMover
# 101 remove editor access

## Issue
Resolves 101 

## What was done
- [x] Add removeUser from UserSpaceMapping on backend
- [x] Add remove user capability on frontend 

## How to test
1. Open space
2. Change address, appending `#perm` to end of address to activate feature flag
3. Invite a user to the space
4. Go back to invite user to space. The user should appear. 
5. On the user's permission, open the submenu. Editor should be checked. Clicking on it does nothing (closes submenu)
6. On the user's permission, open the submenu. Remove should be selectable. Clicking on it removes the user from the space.


## Accessiblity Criteria
